client/packages/invoices/src/InboundShipment/DetailView/Footer/Footer.tsx

```diff
@@ -78,7 +78,11 @@ export const FooterComponent: FC<FooterComponentProps> = ({
       icon: <DeleteIcon />,
       onClick: onDelete,
       disabled: isDisabled || !isManuallyCreated,
-      disabledToastMessage: t('messages.cant-delete-transferred'),
+      disabledToastMessage: !isManuallyCreated
+        ? t('messages.cant-delete-transferred')
+        : isDisabled
+          ? t('messages.cant-delete-generic')
+          : undefined,
```

Comment:
Nice! NIT - this probably doesn't need the second ternary - the message won't display unless one of those disabled states is true, so we don't really need the message to be undefined.

```suggestion
      disabledToastMessage: !isManuallyCreated
        ? t('messages.cant-delete-transferred')
        : t('messages.cant-delete-generic')
```

Reply:
Updated here 2b8a7b993b8799ee0aa06e6e27be1398942eb392 😄